router2/src/dml_handlers/instrumentation.rs

```diff
@@ -205,10 +206,14 @@ mod tests {
         );
     }
 
+    fn summary() -> WriteSummary {
+        WriteSummary::new(vec![])
```

Comment:
```suggestion
        WriteSummary::default()
```

Or could inline into the one use 👍